.pre-commit-config.yaml

```diff
@@ -12,7 +12,6 @@ repos:
     hooks:
       - id: go-fmt
       - id: go-vet
-      - id: go-imports
```

Comment:
go-imports doesn't seem to support generics.
This means the tool is not maintained. So better not use it anyways.

Reply:
the goimports on my machine was outdated...